improve the usage section of the README (#6041)
dani-garcia/vaultwarden: README.md

> That means it will only work via `http://localhost:8000` (using the port from the example below) or if you [enable HTTPS](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS).

The recommended way to install and use Vaultwarden is via our container images which are published to [ghcr.io](https://github.com/dani-garcia/vaultwarden/pkgs/container/vaultwarden), [docker.io](https://hub.docker.com/r/vaultwarden/server) and [quay.io](https://quay.io/repository/vaultwarden/server).
See [which container image to use](https://github.com/dani-garcia/vaultwarden/wiki/Which-container-image-to-use) for an explanation of the provided tags.

There are also [community driven packages](https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages) which can be used, but those might be lagging behind the latest version or might deviate in the way Vaultwarden is configured, as described in our [Wiki](https://github.com/dani-garcia/vaultwarden/wiki).

Alternatively, you can also [build Vaultwarden](https://github.com/dani-garcia/vaultwarden/wiki/Building-binary) yourself.

> While Vaultwarden is based upon the [Rocket web framework](https://rocket.rs) which has built-in support for TLS our recommendation would be that you setup a reverse proxy (see [proxy examples](https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples).

Reviewer: ```suggestion
While Vaultwarden is based upon the [Rocket web framework](https://rocket.rs) which has built-in support for TLS our recommendation is to setup a reverse proxy (see [proxy examples](https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples).
```

Our recommendation "is".... "would be" is too weak and slightly awkward. If you check other "recommendation" sentences for references, you will see that "would be" is not used unless required by conditionals.

Author: I disagree. "My recommendation would be" is a perfectly fine way to politely suggest something.

  --env DOMAIN="https://vw.domain.tld" \
  --volume /vw-data/:/data/ \
  --restart unless-stopped \

>   --publish 127.0.0.1:8000:80 \

Reviewer 2: This is somewhat unrelated to this, but if we're trying to simplify instructions, docker run has a `--pull=always` flag, which means we can remove the previous `docker pull` instruction and keep this as a single command:
https://docs.docker.com/reference/cli/docker/container/run/#pull

Author: Oh, `podman` has [that flag](https://docs.podman.io/en/v5.0.1/markdown/podman-run.1.html#pull-policy) too, so I think that it should be fine to add this.
